lib/menubar/init.lua

```diff
@@ -119,10 +119,9 @@ menubar.right_label = "▶▶"
 -- @tfield[opt="◀◀"] string left_label
 menubar.left_label = "◀◀"
 
--- awful.widget.common.list_update adds three times a margin of dpi(4)
--- for each item:
--- @tfield number list_interspace
-local list_interspace = theme.xresources.apply_dpi(4) * 3
+-- awful.widget.common.list_update adds spacing of dpi(4) between items.
+-- @tfield number list_spacing
+local list_spacing = theme.xresources.apply_dpi(4)
```

Comment:
Do you think it would make sense to expose this field?

Reply:
oh, wait, i double-checked, actually that spacing is hardcoded in awful.widgetcommon.list_update(), so here this variable only reflects that hardcoded value of common.list_update

Reply:
For future references:

https://github.com/awesomeWM/awesome/blob/master/lib/awful/widget/common.lua#L82
https://github.com/awesomeWM/awesome/blob/master/lib/awful/widget/common.lua#L88
https://github.com/awesomeWM/awesome/blob/master/lib/awful/widget/common.lua#L89